deps/build.jl

```diff
@@ -23,5 +23,5 @@ if !isfile(tagfile) || readchomp(tagfile) != cubvers
             run(`gcc -shared -o ../libcubature.so hcubature.o pcubature.o`)
         end
     end
-    run(`echo $cubvers` |> tagfile)
+    @compat run(pipeline(`echo $cubvers`, tagfile))
```

Comment:
You don't need `@compat` -- that macro is only for syntax changes.  `pipeline` is an ordinary function, and the Compat macro just exports a `pipeline(...)` replacement function in 0.3.